queries.sql

```diff
@@ -9,4 +9,49 @@ SELECT date_of_birth FROM animals WHERE name IN('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT name, neutered FROM animals WHERE neutered = True;
 SELECT name FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE animals.name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE animals.name NOT LIKE '%mon';
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+TABLE animals;
+
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01;
+SAVEPOINT SP1;
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+ROLLBACK SP1;
+UPDATE animals
+SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+-- aggregate
+
+SELECT COUNT(name) FROM animals;
```

Comment:
- [ ] After each transaction, please take a screenshot and then add them to the PR summary for further verification 😃 👍